jira/internal/watcher_impl.go

```diff
@@ -42,8 +42,8 @@ func (w *WatcherService) Gets(ctx context.Context, issueKeyOrID string) (*model.
 // POST /rest/api/{2-3}/issue/{issueKeyOrID}/watchers
 //
 // https://docs.go-atlassian.io/jira-software-cloud/issues/watcher#add-watcher
-func (w *WatcherService) Add(ctx context.Context, issueKeyOrID string) (*model.ResponseScheme, error) {
-	return w.internalClient.Add(ctx, issueKeyOrID)
+func (w *WatcherService) Add(ctx context.Context, issueKeyOrID, accountId string) (*model.ResponseScheme, error) {
+	return w.internalClient.Add(ctx, issueKeyOrID, accountId)
```

Comment:
```suggestion
func (w *WatcherService) Add(ctx context.Context, issueKeyOrID, accountID string) (*model.ResponseScheme, error) {
	return w.internalClient.Add(ctx, issueKeyOrID, accountID)
```

Reply:
why these?, this would not be a consistent naming

Reply:
https://google.github.io/styleguide/go/decisions.html#initialisms
This should explain why i suggested those changes.
I am currently trying to fix all existing case issues in the repo, but since there are many it takes a while, so i would like to ensure they are correct when adding new code.